applications/luci-app-transmission/po/ru/transmission.po

```diff
@@ -1,67 +1,67 @@
 msgid ""
 msgstr ""
+"Content-Type: text/plain; charset=UTF-8\n"
 "Project-Id-Version: LuCI: transmission\n"
-"Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2009-05-19 19:36+0200\n"
-"PO-Revision-Date: 2012-08-15 17:35+0300\n"
-"Last-Translator: Roman A. aka BasicXP <x12ozmouse@ya.ru>\n"
-"Language-Team: Russian <x12ozmouse@ya.ru>\n"
-"Language: ru\n"
+"POT-Creation-Date: 2012-08-15 17:35+0300\n"
+"PO-Revision-Date: 2018-01-31 18:50+0300\n"
+"Language-Team: http://cyber-place.ru\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%"
-"10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
-"X-Generator: Pootle 2.0.6\n"
-"X-Poedit-SourceCharset: UTF-8\n"
+"X-Generator: Poedit 1.8.7.1\n"
+"Last-Translator: Vladimir aka sunny <picfun@ya.ru>\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
+"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
+"Language: ru\n"
+"Project-Info: Это технический перевод, не дословный. Главное-удобный русский "
+"интерфейс, все проверялось в графическом режиме, совместим с другими apps\n"
 
 msgid "Alternative download speed"
-msgstr "Альтернативная скорость загрузки"
+msgstr "Альтернативное ограничение скорости загрузки"
 
 msgid "Alternative speed enabled"
-msgstr "Альтернативная скорость включена"
+msgstr "Альтернативное ограничение скорости включено"
 
 msgid "Alternative speed time begin"
-msgstr "Время начала альтернативной скорости"
+msgstr "Время запуска альтернативного ограничения скорости"
 
 msgid "Alternative speed time day"
-msgstr "День альтернативной скорости"
+msgstr "Время суток альтернативного ограничения скорости"
```

Comment:
@hnyman  At this point I've tired of looking further and think that this PR shouldn't be merged, while there are some good points but in the most part it's a technical translation.
I suppose PR author should check each translation entry manually instead of automatic translation without any check. Some entries do not even correspond to its original meaning.

Reply:
"Some entries do not even correspond to its original meaning". - It's a lie..
Хотите переводить, преводите, берете 52 приложения, делаете единый шаблон перевода, чтобы одни и те же фразы быи переведены одинаково и вперед со скиншотами пожалуйста:
http://cyber-place.ru/showthread.php?t=2620

Reply:
Перевод в процессе правки, хотите помогать или возглавить процесс ради бога, только мешать не надо...